plugins/info/include/plugins/info/info.h

```diff
@@ -81,15 +81,22 @@ class Info : public PluginBase {
         char product_name[32]; /**< @brief Name of product. */
     };
 
+    /**
+     * @brief Type containing identification.
+     */
+    struct Identification {
+        uint8_t hardware_uuid[18]; /**< @brief UUID of hardware. */
```

Comment:
can we call this uid instead of uuid?
UUID is generally understood to be a 16 byte identifier: https://en.wikipedia.org/wiki/UUID

Reply:
Hm, so what is it actually?
UUID: universally unique ID
UID: unique ID

According to the MAVLink spec it's a UID, so I guess you're right.
https://mavlink.io/en/messages/common.html#AUTOPILOT_VERSION